easybuild/easyconfigs/r/Relate/Relate-20211123-foss-2020b-R-4.0.3.eb

```diff
@@ -0,0 +1,68 @@
+# Author: Jasper Grimm (UoY)
+easyblock = 'CMakeMake'
+
+name = 'Relate'
+version = '20211123'
+local_commit = '78964f4'
+versionsuffix = '-R-%(rver)s'
+
+homepage = 'https://myersgroup.github.io/relate/'
+description = "Software for estimating genome-wide genealogies for thousands of samples"
+citing = """
+[1] Leo Speidel, Marie Forest, Sinan Shi, Simon Myers. A method for estimating genome-wide
+    genealogies for thousands of samples. Nature Genetics 51: 1321-1329, 2019.
+[2] Leo Speidel, Lara Cassidy, Robert W. Davies, Garrett Hellenthal, Pontus Skoglund, Simon Myers.
+    Inferring population histories for ancient genomes using genome-wide genealogies.
+    bioRxiv:2021.02.17.431573.
+"""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+github_account = 'MyersGroup'
+source_urls = [GITHUB_LOWER_SOURCE]
+sources = ['%s.tar.gz' % local_commit]
+patches = ['%(name)s-20211123_add-cmake-install-targets.patch']
+checksums = [
+    'cc19a50036f78ede9f4d52cc0cbb81d90d1bdb51f8b70a1f001a4f32dd26dbd0',  # 78964f4.tar.gz
+    # Relate-20211123_add-cmake-install-targets.patch
+    'd4565aeef0036013b82b37e00909965a6447ab14b154caa3e304307d6f7818e4',
+]
+
+builddependencies = [('CMake', '3.18.4')]
+
+dependencies = [
+    ('zlib', '1.2.11'),
+    ('R', '4.0.3'),     # required for some analysis scripts
+]
+
+runtest = 'test'
+
+# script directories to add to path:
+local_scripts = ['DetectSelection', 'EstimatePopulationSize', 'PrepareInputFiles', 'RelateParallel',
+                 'SampleBranchLengths', 'TreeView']
+# un/comment as appropriate:
+# local_scripts += ['RelateLSF']
+# local_scripts += ['EstimatePopulationSizeSGE', 'RelateSGE']
+# local_scripts += ['RelateSlurm']
```

Comment:
hm... so, if I understand correctly, they can't all be added to the path because although the actual Relate*.sh scripts inside them include the scheduler in the filename, the other scripts have the same name and conflict...

besides the comment for whoever is installing, what about using `modloadmsg` to tell the users about these folders in the generic case?

Reply:
yeah annoyingly they share names whilst being unique, so it's probably best to avoid throwing them all in $PATH.